napari/_qt/_qapp_model/qactions/_file.py

```diff
@@ -0,0 +1,169 @@
+    # Action(
+    #     id=CommandId.DLG_SAVE_LAYERS,
+    #     title=CommandId.DLG_SAVE_LAYERS.title,
+    #     callback=QtViewer._save_layers_dialog,
+    #     kwargs={'selected': True},  # <<<<< TODO!
+    #     menus=[{'id': MenuId.MENUBAR_FILE, 'group': '3_'}],
+    #     keybindings=[StandardKeyBinding.Save],
+    #     enablement='num_selected_layers > 0'
+    # ),
+    Action(
+        id=CommandId.DLG_SAVE_LAYERS,
+        title=trans._('Save All Layers...'),
+        callback=QtViewer._save_layers_dialog,
+        menus=[{'id': MenuId.MENUBAR_FILE, 'group': MenuGroup.SAVE}],
+        # Conflict: ctrl+shift+s typically means "save as ..."
+        keybindings=[
+            {'primary': KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KeyS}
+        ],
+        enablement=(LLCK.num_selected_layers > 1),
+    ),
+    Action(
+        id=CommandId.DLG_SAVE_CANVAS_SCREENSHOT,
+        title=CommandId.DLG_SAVE_CANVAS_SCREENSHOT.title,
+        callback=QtViewer._screenshot_dialog,
+        menus=[{'id': MenuId.MENUBAR_FILE, 'group': MenuGroup.SAVE}],
+        keybindings=[{'primary': KeyMod.Alt | KeyCode.KeyS}],
+        status_tip=trans._('Save screenshot of current display, default .png'),
+    ),
+    Action(
+        id=CommandId.DLG_SAVE_VIEWER_SCREENSHOT,
+        title=CommandId.DLG_SAVE_VIEWER_SCREENSHOT.title,
+        callback=Window._screenshot_dialog,
+        menus=[{'id': MenuId.MENUBAR_FILE, 'group': MenuGroup.SAVE}],
+        keybindings=[{'primary': KeyMod.Alt | KeyMod.Shift | KeyCode.KeyS}],
+        status_tip=trans._('Save screenshot of current display, default .png'),
+    ),
+    Action(
+        id=CommandId.COPY_CANVAS_SCREENSHOT,
+        title=CommandId.COPY_CANVAS_SCREENSHOT.title,
+        callback=QtViewer.clipboard,
+        menus=[{'id': MenuId.MENUBAR_FILE, 'group': MenuGroup.SAVE}],
+        keybindings=[{'primary': KeyMod.Alt | KeyCode.KeyC}],
+        status_tip=trans._(
+            'Copy screenshot of current display to the clipboard'
+        ),
+    ),
+    Action(
+        id=CommandId.COPY_VIEWER_SCREENSHOT,
+        title=CommandId.COPY_VIEWER_SCREENSHOT.title,
+        callback=Window.clipboard,
+        menus=[{'id': MenuId.MENUBAR_FILE, 'group': MenuGroup.SAVE}],
+        keybindings=[{'primary': KeyMod.Alt | KeyMod.Shift | KeyCode.KeyC}],
+        status_tip=trans._(
+            'Copy screenshot of current display with the viewer to the clipboard'
+        ),
+    ),
+    Action(
+        id=CommandId.DLG_CLOSE,
+        title=CommandId.DLG_CLOSE.title,
+        callback=Window._close_dialog,
+        menus=[{'id': MenuId.MENUBAR_FILE, 'group': MenuGroup.CLOSE}],
+        keybindings=[StandardKeyBinding.Close],
+    ),
+    Action(
+        id=CommandId.RESTART,
+        title=CommandId.RESTART.title,
+        callback=_restart,
+        menus=[
+            {
+                'id': MenuId.MENUBAR_FILE,
+                'group': MenuGroup.CLOSE,
+                'when': running_as_bundled_app(),
```

Comment:
I see:

https://github.com/napari/napari/blob/e1415c59ca3a1de78ac7e31784edd46a04155b9e/napari/utils/misc.py#L53-L58

I assume it is still okay to use this function until then? Should a note be added?
cc @DragaDoncila